### DISCUSSION TITLE
Welcome to node-red-contrib-whatsapp-link Discussions!

### DISCUSSION THREAD

**Discussion by raweee** [raweee/node-red-contrib-whatsapp-link#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TVEN2** [comment]
Hi,
I'm using the "group-out" node to send a whatsapp message to a group, this works fine. However, it does send automatically a message to other groups when there is a change within that partical group (e.g. some member is added). The message contains something like "!node-red", any clue on how to disable this?
Greets,
Ties

**Comment by javidombla** [comment]
I have this error when trying to install on node-red:
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -2
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

Any help, thanks

**Reply by gerbuuu** [reply]
try first to update everything, thats what I did. when i had errors.

**Reply by pwbecker** [reply]
> I have this error when trying to install on node-red: npm ERR! code ENOENT npm ERR! syscall spawn git npm ERR! path git npm ERR! errno -2 npm ERR! enoent An unknown git error occurred npm ERR! enoent This is related to npm not being able to find a file. npm ERR! enoent
> 
> Any help, thanks

Assuming you do have git installed, this is because you are using the snap version of node-red. It cannot access git.
To fix, run:
# snap remove node-red
# apt install npm 
# npm install -g --unsafe-perm node-red

**Reply by javidombla** [reply]
Thanks for your reply, I managed to install it, but it only worked momentarily, now I only get an error in node-red.
29 Jan 19:17:09 - [info] Server now running at http://127.0.0.1:1880/
29 Jan 19:17:09 - [info] Starting flows
29 Jan 19:17:11 - [info] [whatsappLink:Web] Status : Initializing Whatsapp..
29 Jan 19:17:11 - [info] Started flows
29 Jan 19:17:21 - [red] Uncaught Exception:
29 Jan 19:17:21 - [error] TypeError: Cannot read properties of null (reading '1')
    at LocalWebCache.persist (C:\Users\javi\.node-red\node_modules\whatsapp-web.js\src\webCache\LocalWebCache.js:34:69)
    at C:\Users\javi\.node-red\node_modules\whatsapp-web.js\src\Client.js:744:36
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
